lib/config/folderconfiguration.go

```diff
@@ -88,7 +92,76 @@ func (f *FolderConfiguration) CreateMarker() error {
 }
 
 func (f *FolderConfiguration) HasMarker() bool {
-	_, err := os.Stat(filepath.Join(f.Path(), ".stfolder"))
+	_, err := os.Stat(f.SyncthingPath())
+	if err != nil {
+		return false
+	}
+	return true
+}
+
+func (f *FolderConfiguration) CreateDefaultIgnores() error {
+	if f.HasIgnores() {
+		l.Infoln("Not overriding existing ignores with default ignores")
+		return nil
+	}
+
+	fd, err := os.Create(filepath.Join(f.SyncthingPath(), "ignores.txt"))
+	if err != nil {
+		return err
+	}
+
+	ignores := "#include .stsharedignore"
+	if runtime.GOOS == "windows" {
+		ignores = ignores + `
+ehthumbs.db
```

Comment:
These lists should probably live as constants in a separate file for the purpose